br/pkg/backup/push.go

```diff
@@ -51,6 +57,20 @@ func newPushDown(mgr ClientMgr, capacity int) *pushDown {
 	}
 }
 
+func checkStoreLiveness(s *metapb.Store) error {
+	if s.State != metapb.StoreState_Up {
+		return errors.Annotatef(berrors.ErrKVStorage, "the store state isn't up, it is %s", s.State)
+	}
+	// If the field isn't present (the default value), skip this check.
+	if s.GetLastHeartbeat() > 0 {
+		lastHeartBeat := time.Unix(0, s.GetLastHeartbeat())
+		if sinceLastHB := time.Since(lastHeartBeat); sinceLastHB > storeDisconnectionDuration {
+			return errors.Annotatef(berrors.ErrKVStorage, "the store last heartbeat is too long ago, it is %s", sinceLastHB)
```

Comment:
how about logs the store id or more information?

Reply:
This information would be printed in the outside, like:
<img width="1770" alt="image" src="https://user-images.githubusercontent.com/36239017/232456197-645cb76d-0885-4556-9300-168c84f61f52.png">